Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -546,8 +585,13 @@ def list_asset_internet_exposure_command(client: Client, args: dict[str, Any]) -
         search_params.append({"field": "type", "operator": "in", "value": [asm_type]})
     if has_active_external_services:
         search_params.append({"field": "has_active_external_services", "operator": "in", "value": [has_active_external_services]})
+    if search_from:
+        search_from = int(search_from)
```

Comment:
I would move this up to where the variable is instantiated, since you will always have a default.